pebblo/entity_classifier/utils/config.py

```diff
@@ -38,6 +38,33 @@ class SecretEntities(Enum):
     GOOGLE_API_KEY = "google-api-key"
 
 
+class PIIGroups(Enum):
+    Identification = "pii-identification"
+    Financial = "pii-financial"
+    Secrets = "secrets_and_tokens"
+
+
+entity_group_conf_mapping = {
+    # Identification
+    Entities.US_SSN.value: (0.8),
```

Comment:
Set may not be good choice if we add multiple fields per entity in future. With current thinking in mind - to store per entity confidence score and group, this would suffice though.